EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+In the future (it should be a separate EIP) we can switch to some more sophisticated DeFi scheme (`SalaryWithDAO` contract does support donations in DeFi token) of converting ETH to ERC-1155 tokens to be transferred to `SalaryWithDAO` than simply wrapping ETH into ERC-1155 token by `DonateETH`, to increase future value of the tokens transferred to `SalaryWithDAO`.
+
+Additional proposal for future consideration:
+
+- Consider (by the DAO voting) "obliging" the above mentioned DAO by a smart contract (with ownership transfer) to set the minimum oracle finish date in the future to some fixed value (need to discuss the exact number of years to lock funds for).
+- Consider (in a separate EIP) creating new oracles with shifted to father future (e.g. by one year forward once per year) the minimum finish times and spreading the transferred funds between several such oracles.
+
+## Rationale
+The Future Salaries is the _only_ known system of distributing significant funds to common good producers. (Quadratic funding aimed to do a similar thing, but in practice as we see on GitCoin it favors a few developers, ignores project of highly advanced scientific research that is hard to explain to an average developer, and encourages colluding, and it just highly random due to small number of donors. Also quadratic funding simply does not gather enough funds to cover common good needs). So this EIP is the only known way to recover the economy.
+
+The economical model of Future Salaries is described in [this research article preprint](https://github.com/vporton/gitcoin-web/blob/future/app/assets/docs/science-salaries.pdf).
+
+Funding multiple oracles with different finish time would alleviate the future trouble that the ETH (or other tokens) supply would suddenly increase when the oracle finishes.
+
+## Backwards Compatibility
+This EIP requires a hard fork, so we need some time for all clients to be updated. The change in clients is however trivial and the update won't require much time.
+
+## Security Considerations
+The security considerations are:
+- The contract audit of `SalaryWithDAO` and of `DefaultDAOInterface` was paid for. But the re-audit would be good for such a drastic change.
+- `DonateETH` also needs to be audited.
+- The value of wrapped ETH transferred to `SalaryWithDAO` may happen to decrease with time (despite that this EIP itself and some others aim to increase ETH value), so (as it were told to be discussed in a separate EIP) need to consider alternative DeFi schemes than simply wrapping ETH in ERC-1155.
+- Future Salaries system is expected to create big unfounded bubbles of somebody's personal tokens. This problem seems inevitable, but the existing problems this EIP aims to solve are much worse.
+- The DAO may switch to a non-effective or biased way of voting (for example to being controlled by one human) thus distributing funds unfairly. This problem could be solved by a future fork of Ethereum that would "confiscate" funds from the DAO.
```

Comment:
This section is for describing potential attack vectors and gotchas people will need to consider when implementing this EIP.  As above, it should not be a todo list but instead should be written in its final form targeting a reader 5-10 years from now who is reading this EIP and wants to implement it.  It also generally isn't the right place for discussion about monetary policy risk and should instead focus solely on technical risks.